changelogs/fragments/2008-update-java-cert-replace-cert-when-changed.yml

```
@@ -0,0 +1,7 @@
+minor_changes:
+  - "Java_cert: Change `state: present` to check certificates by hash, not just alias name (https://github.com/ansible/ansible/issues/43249)"
```

```suggestion
  - "java_cert - change ``state: present`` to check certificates by hash, not just alias name (https://github.com/ansible/ansible/issues/43249)."
```
The other lines should be similar.